client/state/jetpack-settings/settings/test/reducer.js

```diff
@@ -84,9 +84,9 @@ describe( 'reducer', () => {
 					12345678: SETTINGS_FIXTURE[ siteId ]
 				},
 				action = {
-					type: JETPACK_SETTINGS_RECEIVE,
```

Comment:
Either that or I totally misparsed that test's description should update settings in the items object